src/shared/services/form/form-value.service.ts

```diff
@@ -69,6 +69,12 @@ export class FormValueService {
     Object.keys(cloneForm['data']).forEach((key) => {
       if (caseFields.findIndex((element) => element.id === key) < 0) {
         delete cloneForm['data'][key];
+      } else {
+        caseFields.forEach(function (e) {
+          if (e.id === key && e.field_type && e.field_type.type === 'DynamicList') {
```

Comment:
I don't think `e.field_type` null check is required because field always has  a field_type

Reply:
removed